dev/Debian9_qt4/Dockerfile

```diff
@@ -1,37 +1,48 @@
 FROM debian:stretch
```

Comment:
On this base image, I would install only the dependencies according to the documentation:
https://github.com/mxcube/mxcube/blob/master/docs/source/installation_instructions_qt4.rst

Reply:
OK, that makes sense. 

I shall have to wait a week to do it, though, since I need to prepare for our yearly workshop with the (paying) consortium members (that starts on Monday). I had not realised that all these things, including the installation instructions need to be in sync. And since I have had lots of trouble in getting these Dockerfiles to work exactly, I need time to retest everything, and probably modify the installation_instructions. So next week it is.

I shall leave the PR up in the meantime, in case anybody wants a look.

Thanks for the improvements,

Rasmus

Reply:
@jordiandreu 
Hi, I am looking at cleaning up this on my side - and shall retire my PR shortly.

Meanwhile one question, in the interest of clean-up: 
Do you actually need pymca and enum34 in the Dockerfile? I do not have them in my version.

Rasmus

Reply:
Hi @rhfogh ,
I'm sure you  need enum34 in master branch since I found this dependency not so long time ago.
regarding the pymca, I haven tried without (maybe is required for the fluorescene scans) but it is also among the Qt4 dependencies list:

https://github.com/mxcube/mxcube/blob/master/docs/source/installation_instructions_qt4.rst